servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/publisher/TimeoutPublisherTest.java

```diff
@@ -68,13 +76,26 @@
     private final TestPublisher<Integer> publisher = new TestPublisher<>();
     private final TestPublisherSubscriber<Integer> subscriber = new TestPublisherSubscriber<>();
     private final TestSubscription subscription = new TestSubscription();
+    private final boolean untilTermination;
     private TestExecutor testExecutor;
 
+    public TimeoutPublisherTest(boolean untilTermination) {
+        this.untilTermination = untilTermination;
+    }
+
     @Before
     public void setup() {
         testExecutor = executorRule.executor();
     }
 
+    @Parameterized.Parameters(name = "{index} - {0}")
```

Comment:
Consider clarifying what the boolean means: `"{index} - reset timeout at onNext: {0}"`